nx/source/services/vi.c

```diff
@@ -18,14 +17,23 @@ static Service g_viISystemDisplayService;
 static Service g_viIManagerDisplayService;
 static Service g_viIHOSBinderDriverIndirect;
 
-static Result _viGetSession(Service* srv, Service* srv_out, void* inraw, size_t rawsize);
-static Result _viGetSessionNoParams(Service* srv, Service* srv_out, u64 cmd_id);
+NX_INLINE Result _viGetSession(Service* srv, Service* srv_out, u32 cmd_id, u32 inval) {
```

Comment:
Keep this static (non-inline), cmd_in param should be the last

Reply:
_viCmdGetSession